docs/rfc/rfc-1.adoc

```diff
@@ -458,11 +460,31 @@ To make sure that older wallets are still accessible for redemption, we need to
 perform heartbeats. The signing group signs a block when block count mod
 `heartbeat_block_length` = 0 and then does _not_ publish the result. If there
 are ever less than `heartbeat` operators that participate in the heartbeat, the
-ones that did can create and sign a new transaction that lists the _inactive_
-operators. Once this transaction is posted to the ethereum chain, we can
-iterate through the inactive operators, remove them from reward eligibility
-for `failed_heartbeat_reward_removal_period` amount of time, transfer the remaining
-BTC from the wallet to another random wallet and close this wallet.
+active operators record the inactive operators. If this happens a number of
+times >= `consecutive_failed_heartbeats`, the operators take a union the
+recorded _inactive_ operators, and an operator from among the active operators
+posts a transaction to disable those inactive operators from receiving rewards
+for `failed_heartbeat_reward_removal_period` amount of time. The active
```

Comment:
This operator will be expensive, I expect 1M+ here. Marking operators as ineligible for rewards is expensive and we will have to check threshold number of ECDSA signatures on this request. Do we want to reimburse the caller from the pool of fees?

Reply:
Yeah - I'll point to the `transaction-incentives` section

Reply:
addressed in e28babc